frontend/package.json

```diff
@@ -254,6 +256,7 @@
       ".stories.tsx",
       "<rootDir>/src/app/testQueue/constants.ts",
       "<rootDir>/src/patientApp/timeOfTest/constants.ts"
-    ]
+    ],
+      "transformIgnorePatterns": ["node_modules/(?!axios)/"]
```

Comment:
Do we need this? 🤔 I ran jest without this and it looked to work for me and I don't think we are importing axios anywhere

Reply:
This was for the updated version of smartystreets-javascript-sdk, its no longer needed since we aren't updating that package in this PR. I removed it